test/bridge/SynapseBridgeAvax.ts

```diff
@@ -0,0 +1,125 @@
+import { Signer } from "ethers"
+import { getCurrentBlockTimestamp } from "./testUtils"
+import { solidity } from "ethereum-waffle"
+import { deployments, ethers } from "hardhat"
+
+import chai from "chai"
+import { GenericERC20, RateLimiter, SynapseBridge } from "../../build/typechain"
+import { keccak256 } from "ethers/lib/utils"
+import { randomBytes } from "crypto"
+import { forkChain, MAX_UINT256 } from "../utils"
+import { deployRateLimiter, setupForkedBridge } from "./utilities/bridge"
+import { getBigNumber } from "./utilities"
+
+chai.use(solidity)
+const { expect } = chai
+
+describe("SynapseBridgeAvax", async () => {
```

Comment:
Should either expand the scope of tests or deprecate this file, so that we don't have creep of tests w/o specific purpose.

Reply:
killed